src/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+type ButtonProps = {
+  name: string;
+  content: string;
```

Comment:
you can just use the children prop 

https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children